guide/demos.md

```diff
@@ -1,2 +1,31 @@
+| --------------------------- | -------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `webrtc_videoroom`      | Basic example of [Membrane RTC Engine](https://github.com/membraneframework/membrane_rtc_engine.git). It's as simple as possible just to show you how to use our API.                                                                        | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_videoroom)                                                                                                                                                                                                                                                                                                              |
+| `webrtc_simple`      | Simple example of signaling server based on [Membrane.WebRTC.Server](https://github.com/membraneframework/webrtc-server.git). | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_simple)                                                                                                                                                                                                                                                                                                              |
+| `webrtc_athentication`      | An example of signaling server with authentication based on [Membrane.WebRTC.Server](https://github.com/membraneframework/webrtc-server.git).                                                                        | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_authentication)                                                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| `webrtc_authentication`      | An example of signalling server with authentication based on [Membrane.WebRTC.Server](https://github.com/membraneframework/webrtc-server.git).                                                                        | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_demo/tree/master/webrtc_authentication)                                                                                                                                                                                                                                                                                                              |
```

Reply:
I would generally pass the whole file through grammarly :D

Reply:
Huh, you are right https://sapling.ai/usage/signaling-vs-signalling